packages/ocaml-migrate-parsetree/ocaml-migrate-parsetree.1.3.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "frederic.bour@lakaban.net"
+authors: [
+  "Frédéric Bour <frederic.bour@lakaban.net>"
+  "Jérémie Dimino <jeremie@dimino.org>"
+]
+license: "LGPL-2.1"
+homepage: "https://github.com/ocaml-ppx/ocaml-migrate-parsetree"
+bug-reports: "https://github.com/ocaml-ppx/ocaml-migrate-parsetree/issues"
+dev-repo: "git+https://github.com/ocaml-ppx/ocaml-migrate-parsetree.git"
+doc: "https://ocaml-ppx.github.io/ocaml-migrate-parsetree/"
+tags: [ "syntax" "org:ocamllabs" ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "result"
+  "ppx_derivers"
+  "dune" {build & >= "1.6.0"}
+  "ocaml" {>= "4.02.3" & < "4.09.0"}
```

Comment:
why are those bound here if the only compiler it can be installed with is `4.08.0+beta3`?

Reply:
issue fixed.

Reply:
I think this version is pretty close to the "final" one,
so I thought we would just drop the line (the `conflicts`
after your change) once 4.08.0 is out.

Reply:
I don't think it costs us anything to allow it already. Also it allows people using other variants of `ocaml.4.08.0+beta3` (with `flambda` or whatever) to use it out of the box